internal/internal_nexus_task_handler.go

```diff
@@ -162,11 +169,23 @@ func (h *nexusTaskHandler) handleStartOperation(ctx context.Context, nctx *Nexus
 	if callbackHeader == nil {
 		callbackHeader = make(map[string]string)
 	}
+	var nexusLinks []nexus.Link
+	for _, link := range req.GetLinks() {
+		linkData, err := proto.Marshal(link)
```

Comment:
I think the API can just send a link type that's `type + data` and avoid unmarshalling in the server and then marshaling in the SDK.